packages/react-components/src/components/DevicesButton.tsx

```diff
@@ -268,7 +268,8 @@ export const generateDefaultDeviceMenuProps = (
       // A side effect of this workaround is that the context menu stays open when window is resized, and may
       // get detached from original target visually. That bug is preferable to the bug when this value is not set -
       // The Callout (frequently) gets dismissed automatically.
-      preventDismissOnResize: true
+      preventDismissOnResize: true,
+      preventDismissOnScroll: true
```

Comment:
Looks like this API is now deprecated: https://developer.microsoft.com/en-us/fluentui#/controls/web/callout#ICalloutProps

![image](https://user-images.githubusercontent.com/82062616/178663787-d915ea12-72ac-44b2-9909-71045ba51440.png)

But also note that using `preventDismissOnEvent` will override `preventDismissOnResize` as well, so you'll have to test to make sure that resize behavior remains correct (e.g., add a remote participant while the participant pane flyout is open).